sdk/tests/conformance/resources/gl-object-get-calls.js

```diff
@@ -122,6 +121,11 @@ gl.bindTexture(gl.TEXTURE_2D, null);
 var framebuffer = gl.createFramebuffer();
 gl.bindFramebuffer(gl.FRAMEBUFFER, framebuffer);
 gl.framebufferTexture2D(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT0, gl.TEXTURE_2D, texture, 0);
+var colorAttachmentsNum = 1;
+if (contextVersion > 1) {
+  colorAttachmentsNum = gl.getParameter(gl.MAX_COLOR_ATTACHMENTS);
+  gl.framebufferTexture2D(gl.FRAMEBUFFER, gl.COLOR_ATTACHMENT0 + colorAttachmentsNum - 1, gl.TEXTURE_2D, texture, 0);
```

Comment:
Hi Ken, Thanks for your reviewing. The for loop below also tests these two attachment points, because the step in the loop is MAX_COLOR_ATTACHMENTS - 1, instead of 1. I add a comment before the for loop.